docs/UserGuide.md

```diff
@@ -259,6 +264,7 @@ Examples:
 
 Finds students whose `NAME` contain any of the given keywords.  
 Note that you have to run `list` to display all the students again.
+Note `Students` list in the `classes` tab will only show students found, `size` of the class will still remain constant.
```

Comment:
Maybe u can explain what constant means here. It may be slightly ambiguous. Maybe u can say `remains unchanged despite the filtering` or `is not affected by the filtering`